### PR TITLE
Solution created for NGINX http server

### PR DESCRIPTION
**Pull request by v-laanjana**
Change(s):
See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution Illusive Attack Management System.
See guidance below
Version Updated:
2.0.0
See guidance below
Testing Completed:
The link for test results is
[Sentinel Solution Migration ](https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7BA5557CE3-8CA4-4E71-ADD5-F7622AD64FAA%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20Sentinel%20Solution%20Migration%20-%20TemplateSpec%20-%20NGINX%20HTTP%20Server.docx&action=default&mobileredirect=true&share=IQHjfFWlpIxxTq3V92Iq1k-qAXMWhRSD9jJmxTPBoVKBMvM)NGINX-HTTP-Server

See guidance below
This is an 3P solution.

### CODE REVIEW

File: Solutions/NGINX HTTP Server/Data Connectors/Connector_NGINX_agent.json [hunk 1]
```diff
@@ -2,8 +2,7 @@
     "id": "NGINXHTTPServer",
```
Comment (v-amolpatil) [inline Solutions/NGINX HTTP Server/Data Connectors/Connector_NGINX_agent.json:2]: Please update `isPreview` from `true` to `false`.
Currently, it has below code:
`"availability": {
        "status": 1,
        "isPreview": true
    },`

File: Solutions/NGINX HTTP Server/Data/Solution_Nginx.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-  "WorkbookDescription": "Sets the time name for analysis",
+  "Name": "NGINX HTTP Server",
+  "Author": "Microsoft - support@microsoft.com",
+   "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Azure_Sentinel.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "The [NGINX HTTP Server](https://nginx.org/) data connector provides the capability to ingest [NGINX HTTP Server](https://nginx.org/#basic_http_features) events into Microsoft Sentinel. Refer to [NGINX Logs documentation](https://nginx.org/en/docs/http/ngx_http_log_module.html) for more information. \n \n **Underlying Microsoft Technologies used:**\n\nThis solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs: \n \n a. [Azure Monitor HTTP Data Collector API](https://docs.microsoft.com/azure/azure-monitor/logs/data-collector-api)",
```
Comment (v-amolpatil) [inline Solutions/NGINX HTTP Server/Data/Solution_Nginx.json:5]: hyperlink is only for `NGINX`. Please remove hyperlink for `HTTP Server` as it is not request as per text change document shown below.
image

File: Solutions/NGINX/Package/mainTemplate.json [hunk 3]
```diff
@@ -1,1182 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (v-laanjana) [reply Solutions/NGINX/Package/mainTemplate.json:1]: done

**Review by v-amolpatil — DISMISSED** [review]

File: Solutions/NGINX HTTP Server/Data/Solution_Nginx.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-  "WorkbookDescription": "Sets the time name for analysis",
+  "Name": "NGINX HTTP Server",
+  "Author": "Microsoft - support@microsoft.com",
+   "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Azure_Sentinel.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "The [NGINX(https://nginx.org/) HTTP Server data connector provides the capability to ingest [NGINX HTTP Server](https://nginx.org/#basic_http_features) events into Microsoft Sentinel. Refer to [NGINX Logs documentation](https://nginx.org/en/docs/http/ngx_http_log_module.html) for more information. \n \n **Underlying Microsoft Technologies used:**\n\nThis solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs: \n \n a. [Azure Monitor HTTP Data Collector API](https://docs.microsoft.com/azure/azure-monitor/logs/data-collector-api)",
```
Comment (v-amolpatil) [inline Solutions/NGINX HTTP Server/Data/Solution_Nginx.json:5]: missing square bracket here

**Pull request merged**